pkg/create/templates/test.go

```diff
@@ -43,7 +43,19 @@ container_exists() {
 }
 
 container_ip() {
-  docker inspect --format="{{"{{"}} .NetworkSettings.IPAddress {{"}}"}}" $(cat $cid_file)
+  if (echo "$OSTYPE" | grep -qs 'darwin'); then
```

Comment:
one approach to avoid depending on docker-machine (and supporting Docker for Mac) could be to use socat to forward traffic for you